src/stdweb.rs

```diff
@@ -5,15 +5,15 @@ use std::cell::RefCell;
 use std_web::web::{window, TypedArray};
 use std_web::unstable::TryInto;
 use webgl_stdweb::{
-    WebGL2RenderingContext, WebGLBuffer, WebGLFramebuffer, WebGLProgram, WebGLRenderbuffer,
+    WebGl2RenderingContext, WebGLBuffer, WebGLFramebuffer, WebGLProgram, WebGLRenderbuffer,
```

Comment:
Looks like this should be reverted, see https://travis-ci.org/grovesNL/glow/jobs/631598229#L1081